src/client/components/layout/InstructionalTextBox.tsx

```diff
@@ -0,0 +1,13 @@
+import { fromTheme } from 'mark-one';
+import styled from 'styled-components';
+
+const StyledInstructionalTextBox = styled.div`
```

Comment:
Very minor point, but the name `StyledInstructionalTextBox` kind of implies the existence of a regular `InstructionalTextBox`. If we don't actually need to create two components, can we just drop `Styled` from here?